hvac/v1/__init__.py

```diff
@@ -1199,11 +1199,13 @@ def transit_sign_data(self, name, input_data, key_version=None, algorithm=None,
             params['context'] = context
         if prehashed is not None:
             params['prehashed'] = prehashed
+        if sign_algorithm is not None:
```

Comment:
These conditionals are probably safe to drop now, correct? Since we're supplying a default value for the `sign_algorithm` parameter, someone calling this method would have to explicitly pass in None to end up not meeting this conditional. Also if a caller did pass in None, I would imagine Vault would default to "pss" when receiving that value (but I don't know that for certain). 

How would you feel about simplifying this further and dropping these conditionals all together?